x-pack/legacy/plugins/siem/server/lib/detection_engine/signals_mapping.json

```diff
@@ -1755,7 +1758,7 @@
         "ignore_above": 1024,
         "type": "keyword"
       },
-      "threat": {
+      "threats": {
```

Comment:
This is directly from:
https://github.com/elastic/ecs/blob/master/generated/elasticsearch/7/template.json#L1759

And should be `threat` as this part of the mapping is still part of the copying mechanism.

You don't want to change this to the word `threats` here.